datastream-server-restli/src/test/java/com/linkedin/datastream/server/TestCoordinator.java

```diff
@@ -656,6 +657,86 @@ public void testCoordinationWithStickyMulticastStrategy() throws Exception {
     zkClient.close();
   }
 
+  /**
+   * testCoordinationWithStickyMulticastStrategyAndMaxTaskLimit is a test to verify that assignment cannot complete
+   * when the tasks per instance is exceeded by the configured threshold
```

Comment:
nit:  `when the tasks per instance exceeds the configured threshold`

Reply:
done